source/develop/release-management/features/network/allowExplicitDnsConfiguration.html.md

```diff
@@ -0,0 +1,132 @@
+---
+title: AllowExplicitDnsConfiguration
+category: feature
+authors: mmucha
+wiki_category: Feature
+wiki_title: Features/AllowExplicitDnsConfiguration
+wiki_revision_count: 12
+wiki_last_updated: 2015-10-14
+feature_name: Allow Explicit Dns Configuration
+feature_modules: Networking
+feature_status: Design
+rfe: https://bugzilla.redhat.com/show_bug.cgi?id=1160667
+
+---
+
+# Allow Explicit Dns Configuration
+
+### Owner
+
+*   Name: [ Martin Mucha](User:mmucha)
+*   Email: mmucha@redhat.com
+
+## Summary
+When new host is added to the system, management network will be created for it. Dns configuration for such network will be obtained from resolv.conf file. However admin should be able to specify overriding configuration in WebAdmin. He can do that at several places:
```

Comment:
1. I don't know how to use shorter lines — if I press enter, paragraph breaks into two. Please advise.
2. done
3. I hope it's done (maybe except for screen shot, which are harder to recreate).
4.
5.done
6. This is joke? Discriminating one gender is inappropriate, while discriminate another one is polite?!?? What a time to live in! Is simple 'he/she' sufficient, or shall I, not to offend anyone, enroll all currently "known" genders, which are(if I'm up-to-date): man, woman, not-identifiable, transgender-man, transgender-woman, pending-transgender-man, pending-transgender-woman and finally none.